packages/docusaurus/src/server/__tests__/configValidation.test.ts

```diff
@@ -153,6 +139,16 @@ describe('normalizeConfig', () => {
         ['this/should/work', {too: 'yes'}],
       ],
     ],
+    [
+      'should accept function for plugin', [
+        function(context, params) {}
```

Comment:
```suggestion
        function(_context, _options) {}
```

_ permits to make eslint happy if you don't need to use the param

options is used instead of params